util/src/main/proto/Mcf.proto

```diff
@@ -15,6 +15,7 @@
 syntax = "proto2";
 
 package org.datacommons.proto;
+import "Debug.proto";
```

Comment:
just noticed this - does the [class naming convention](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names) apply to protos as well? this feels very unnatural, and i think we should stick with [proto package convention](https://developers.google.com/protocol-buffers/docs/style#packages)

Reply:
Good catch! I think it should be small case. I will change it in follow-on PR...